### PR TITLE
Updated mongodb models

### PR DESCRIPTION
**Pull request by Kunal22shah**
Added models for  UserProfile, Recruiter and Jobs and updated the user model.

### CODE REVIEW

File: src/server/models/Job.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    },
+    company: {
+        type: String,
+        required: true
+    },
+    category: {
+        type: String,
+        required: true
+    }
+});
```
Comment (Aboudik1452) [inline src/server/models/Job.js:25]: We should add job title in the schema, as many recruiters mention the job title in their job posting on LinkedIn, Indeed, ...

File: src/server/models/UserProfile.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
+const mongoose = require("mongoose");
+
+const Schema = mongoose.Schema;
+
+const userProfileSchema = new mongoose.Schema({
+    user_id: {
+        type: Number,
+        required: true,
```
Comment (Aboudik1452) [inline src/server/models/UserProfile.js:8]: Would the `user_id` in this schema reference the one from the user schema?
I'm not sure where we can connect both of them together.

**Review by Aboudik1452 — CHANGES_REQUESTED** [review]
We should write the back-end tests for uploading a resume, and maybe for creating a recruiter, loading a user profile, and job posting. I suppose we can implement them this sprint.

Other than that, looks good!

File: src/server/models/UserProfile.js (hunk 2 above)
Comment (Resident-png) [reply src/server/models/UserProfile.js:8]: Yeah, I'm not entirely sure either. What would we use the user_id for ? Where/when would it come into play ?

**Comment by imran1289-ah** [comment]
Merging PR as it seems changes were applied

**Pull request merged**